### ISSUE TITLE
blank screen on files tab when using gateway port instead of api port

### ISSUE THREAD

**Issue by swedneck** [ipfs-shipyard/ipfs-webui#1688]
**Describe the bug**

When the address entered for the API port is instead for the gateway port (e.g. http://webui.ipfs.io.ipns.localhost:8080/) the files tab will simply result in a blank page.

**To Reproduce**
1. Go to the settings tab
2. Click on change the api address to the gateway address
3. Go to the files tab
4. See blank screen

**Expected behavior**
It should probably show an error, instead of being blank

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Fedora 32
 - Browser: Firefox 82

**Comment by jessicaschilling** [comment]
Thanks @swedneck - we will look more closely in our next triage. Can you reliably replicate this?

**Comment by swedneck** [comment]
yes, although on double checking reproducibility i noticed that it actually seems to be restricted to subdomain gateway addresses, like `/dns4/webui.ipfs.io.localhost/tcp/8080`. `/dns4/webui.ipfs.io.localhost/tcp/5001` and `/dns4/localhost/tcp/8080` do not result in a blank page.
